File: ProyectoFinCiclo/Python/Ejercicio 3.py
def fibo(n):
    if n>1 :
        return fibo(n-1) + fibo(n-2)
    elif n == 1:
        return 1 
    elif n == 0:
        return 0
    else:
        print("Debe ingresar una posicion mayor a 0")
        return -1 



res = fibo(int(input("Ingrese la posicion de la serie fibo ")))    

print(res)

File: ProyectoFinCiclo/Java/src/proyectofinciclo/Ejercicio3.java
package proyectofinciclo;
import java.util.Scanner;
/**
 *
 * @author Jose Cordova
 */
//Construir una funcion que reciba como parametro un entero y retome ese elemento de la serie de Fibonacci.

public class Ejercicio3 {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        //Declaracion de variables
        int num = 0;
        
        System.out.println("Ingrese la posicion que desea mostrar de la serie fibonacci");
        num = leer.nextInt();
        System.out.println(fibonacci(num));
    }
    
    //Se cre la funcion
    public static int fibonacci(int n){
        if (n>1){
            return fibonacci(n-1) + fibonacci(n-2); 
        }
        else if (n==1) { 
            return 1;
        }
        else if (n==0){ 
            return 0;
        }
        else{ 
            System.out.println("Debes ingresar una posicion mayor o igual a 1");
            return -1; 
        }
    }

}   
        
            
        
            
        
        
        
        
    

        
        